packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/OutputTypes/PropertyDeclaration.cs

```diff
@@ -3,8 +3,45 @@
 
 using System;
 using System.Collections.Generic;
+using Microsoft.Generator.CSharp.Expressions;
+using static Microsoft.Generator.CSharp.Expressions.Snippets;
 
 namespace Microsoft.Generator.CSharp
 {
-    public record PropertyDeclaration(FormattableString? Description, MethodSignatureModifiers Modifiers, CSharpType Type, string Name, PropertyBody Body, IReadOnlyDictionary<CSharpType, FormattableString>? Exceptions = null, CSharpType? ExplicitInterface = null);
+    public record PropertyDeclaration(FormattableString? Description, MethodSignatureModifiers Modifiers, CSharpType Type, string Name, PropertyBody Body, IReadOnlyDictionary<CSharpType, FormattableString>? Exceptions = null, CSharpType? ExplicitInterface = null, string? OriginalDescription = null)
```

Comment:
so i think a long constructor that takes everything might not be good.
How about we define this new property with a `init` accessor so that it does not need to show up here.

Also why we have to need a `OriginalDescription`? would anywhere need this?

Reply:
I added this property since I am building the serialization ctor parameters from the model's properties. Those properties may have formatted descriptions that contain examples for binary data properties. The constructor docs can get very long and unreadable if those long descriptions are added. I've took your first feedback and added the additional property with an init accessor.